biological-journal-of-the-linnean-society.csl

```diff
@@ -26,7 +26,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always" font-weight="bold"/>
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " and="symbol" delimiter-precedes-last="always" font-weight="bold"/>
```

Comment:
you want to change `delimiter-precedes-last="always"` to `delimiter-precedes-last="never"` to prevent commas before the ampersand